File: Python/1046.py
hI, hF = input().split()
hI = int(hI)
hF = int(hF)
if hI > hF: duracao = 24 - (hI - hF)
elif hF > hI: duracao = hF - hI
else: duracao = 24

print("O JOGO DUROU %d HORA(S)" %duracao)

File: C/1046.c
//i, f = inicio e fim
#include <stdio.h>

int main() {
    int i, f, tempo;
    int dia = 24;
    scanf("%d %d", &i, &f);
    if(i == f){
        printf("O JOGO DUROU %d HORA(S)\n", dia);
    }
    else if(i > f){
        tempo = dia-(i-f);
        printf("O JOGO DUROU %d HORA(S)\n", tempo);
    }
    else{
        tempo = f-i;
        printf("O JOGO DUROU %d HORA(S)\n", tempo);
    }
    return 0;
}
